### ISSUE TITLE
ImaPlayer showing toast if ad not found

### ISSUE THREAD

**Issue by marufbd** [kazo0/GoogleMediaFramework-Xamarin#1]
The toast notification on any ad error should be removed from ImaPlayer I guess as it is not usable in production environment.

Ideally I think ImaPlayer should be implemented in C# code using exoplayer and ima sdk

great work! cheers.

**Comment by kazo0** [comment]
Good call, I will change this as well as update the component to include the 0.2.1 release of the framework for Android
